AlertaDengue/dados/views.py

```diff
@@ -400,7 +400,7 @@ def get_context_data(self, **kwargs):
                 # cases estimation
                 cases_est = df_state.casos_est_s.values
 
-                case_series_state[d][s] = cases[:-52]
+                case_series_state[d][s] = cases[-52:]
```

Comment:
the old code is very weird, it looks like the data of the last 12 months was been discarded :O